src/app/pages/rpc-send-transfer/components/rpc-send-transfer.layout.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactNode } from 'react';
+
+import { Stack, StackProps } from '@stacks/ui';
+
+interface SendTransferLayoutProps extends StackProps {
+  children: ReactNode;
+}
+export function RpcSendTransferLayout({ children, ...props }: SendTransferLayoutProps) {
+  return (
+    <Stack
+      alignItems="center"
+      height="100vh"
+      position="relative"
+      maxHeight="calc(100vh - 96px)"
+      spacing="base-loose"
```

Comment:
How certain are we that every child element of this container must have exactly `base-loose` spacing?

Reply:
This was taken from the psbt work > which was taken from the message signing work ...again, these request views need to be consolidated/refactored. I thought it would be better to keep them consistent at least?

Reply:
I refactored here to use a `Flex` instead.